app/controllers/api/v2/puppet_hosts_controller.rb

```diff
@@ -19,7 +19,7 @@ def puppetrun
       end
 
       def fail_and_inform_about_plugin
-        render json: { message: _('To access Puppetrun feature you need to install Foreman Remote Execution Plugin') }, status: :not_implemented
+        render json: { message: _('The Puppetrun feature has been deprecated.') }, status: :not_implemented
```

Comment:
I'm not sure if it's good idea to mention about ability to replace this functionality with REX. What do you think, @ares?

Reply:
I think it is good to mention that there is a replacement. I also think this should not say it's deprecated. Deprecated functionality is still present, but should no longer be used. In this case it has already been removed.

Reply:
We can keep the reference to the plugin, but the message is confusing though, you can't use this endpoint even if you install a REX plugin. If we want to keep it, we need to say something like "you need to use remote execution plugin job to run puppet on demand". I agree we shouldn't say deprecated, instead we should say removed.

Reply:
I see, so something like "The puppetrun feature has been removed, however you can use Remote Execution Plugin to run puppet commands " is more suitable for both of us? @ares @ekohl